core/pom.xml

```diff
@@ -167,6 +167,31 @@
 					</dependency>
 				</dependencies>
 			</plugin>
+			<plugin>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<extensions>true</extensions>
+				<executions>
+					<execution>
+						<id>jar</id>
+						<configuration>
+							<bnd><![CDATA[
+				                Bundle-Copyright: Copyright (c) (2019) Contributors to the Eclipse Foundation.
+				                Git-Descriptor: ${system-allow-fail;git describe --dirty --always --abbrev=9}
```

Comment:
If I understand the result from the issue correctly the result is the following, isn't it?
`Git-Descriptor: 1.1.0-77-g3853a9cff-dirty`

I must confess that I don't find this very useful, especially since you also added `Git-SHA`. Btw. is there a convention for the latter? Because IIRC every tool does it different.

Reply:
We can hold or remove what you need.
I just put in what i like.
So remove?

Reply:
I think less is better. Less to read, understand and maintain. :)
So I would be in favor of remove.

Reply:
removed